.github/workflows/supabase-branch.yml

```
@@ -0,0 +1,309 @@
+          # List available migrations first to verify they can be accessed
+          echo "Listing available migrations:"
+          supabase migration list --experimental || {
+            echo "::error::Failed to list migrations. Verify file structure and Supabase CLI access."
+            exit 1
+          }
+          
+          # Apply migrations with extended error handling
+          echo "Pushing migrations to branch..."
+          DB_PUSH_RESULT=$(supabase db push --experimental 2>&1)
+          PUSH_STATUS=$?
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)